src/components/PDFView/index.js

```diff
@@ -125,6 +128,7 @@ class PDFView extends Component {
                     onLayout={event => this.setState({windowWidth: event.nativeEvent.layout.width})}
                 >
                     <Document
+                        error={<Text style={[styles.textLabel, styles.textLarge]}>{this.props.translate('attachmentView.failedToLoadPDF')}</Text>}
```

Comment:
Could you add testing copy in both languages to the QA steps? I saw you tested yourself :)

Reply:
Will do, I was just trying to escape having to take screenshots of every platform/theme/language combination 🥲

Reply:
You should be good to test only on dark mode for this!